web/templates/dropdown_widget_with_stream_colorblock.hbs

```diff
@@ -1,4 +1,3 @@
 <div id="{{widget_name}}_widget_wrapper" tabindex="0">
-    <div class="stream_header_colorblock"></div>
```

Comment:
Removing this line makes the file name illogical... Consider removing it entirely, and replacing references to it with another file (probably `dropdown_widget.hbs`) as needed. 

This will need careful inspection of all instances of this template in the html, and any related styling (css) and logic (js), to ensure nothing breaks, but all dead code is removed.

Reply:
`dropdown_widget.hbs` already exists. It seems like `dropdown_widget_with_stream_colorblock.hbs` was a wrapper around it. I will try renaming the file to `dropdown_widget_wrapper.hbs` or use a `dropdown_widget.hbs` instead of `dropdown_widget_with_stream_colorblock.hbs`.